h3m/h3mlib/h3m_conversion/convert.c

```diff
@@ -530,6 +530,33 @@ static int _inline_conv_town(uint32_t fm_src,
     return 0;
 }
 
+static int _inline_conv_seers_hut(uint32_t fm_src,
+	struct H3M_OD_BODY_DYNAMIC_SEERS_HUT *body)
```

Comment:
Wrong indentantion but good function, especially a good idea that it can return error if it fails conversion.